packages/debugger/lib/data/selectors/index.js

```diff
@@ -186,6 +186,16 @@ function spoofThisDefinition(contractName, contractId, contractKind) {
 const data = createSelectorTree({
   state: state => state.data,
 
+  /**
+   * data.application
+   */
+  application: {
+    /**
+     * data.application.storageLookup
+     */
+    storageLookup: createLeaf([evm.application.storageLookup], identity)
```

Comment:
does this not belong in `data.views`? I don't know how you do things these days, I guess

Reply:
Man, I dunno, it's all a big mess.  But this selector is going away in the additional change I'm making anyway. :P